presets/wordpress/kool.yml

```diff
@@ -0,0 +1,5 @@
+scripts:
+  php: kool exec app php
```

Comment:
Can we add in `wp` CLI to our base image, and deliver out of the box here the `kool run wp` script with

```yml
    wp: kool exec app wp
```